src/framework/preferences/js/SyllabificationEnactor.js

```diff
@@ -235,66 +242,78 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
                 textStatus,
                 errorThrown
             );
-            // If the pattern file could not be loaded resolve the promise with out a hyphenator (undefined).
+            // If the pattern file could not be loaded, resolve the promise without a hyphenator (undefined).
```

Comment:
This is a comment for the error block on line 239 - note that the signature you attempt to use of (jqXHR, textStatus, errorThrown) cannot be received since the promise rejection API only supplies a single error payload as a value.